### PR TITLE
Issue 4 sign up

### PR DESCRIPTION
**Pull request by yuliia-p**
https://user-images.githubusercontent.com/106217355/190686890-09e41eb4-6a14-496c-aa99-3ce7167387e5.mov

Screen Shot 2022-09-16 at 9 37 05 AM
Screen Shot 2022-09-16 at 9 38 08 AM

Screen Shot 2022-09-16 at 9 37 42 AM

### CODE REVIEW

File: list-of-databases.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    Name     |  Owner   | Encoding | Collate |  Ctype  |   Access privileges   
+-------------+----------+----------+---------+---------+-----------------------
+ dbBookLover | dev      | UTF8     | C.UTF-8 | C.UTF-8 | 
+ dev         | dev      | UTF8     | C.UTF-8 | C.UTF-8 | 
+ postgres    | postgres | UTF8     | C.UTF-8 | C.UTF-8 | 
+ template0   | postgres | UTF8     | C.UTF-8 | C.UTF-8 | =c/postgres          +
+             |          |          |         |         | postgres=CTc/postgres
+ template1   | postgres | UTF8     | C.UTF-8 | C.UTF-8 | =c/postgres          +
+             |          |          |         |         | postgres=CTc/postgres
+(5 rows)
```
Comment (thebearingedge) [inline list-of-databases.txt:11]: You can remove this file

File: client/components/auth-form.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              <input onChange={this.handleChange} type="password" placeholder="At least 8 characters" name="password" id="password" required />
+
+              <div className="button-holder">
+                <button type="submit" className="auth-button">Sign Up</button>
+              <p className='sing-up-text no-margin'>Already have an account?<a className='log-in-a' href="#logn-in">Logn in</a></p>
+              </div>
+          </div>
+        </form>
+      </>
+    );
```
Comment (thebearingedge) [inline client/components/auth-form.jsx:60]: You can remove the react fragment here.

File: client/pages/auth-modal.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import AuthForm from '../components/auth-form';
+
+export default class AuthModal extends React.Component {
+  render() {
+    return (
+      <>
+      <div className='modal'>
+          <AuthForm onAuthClick={this.props.onAuthClick}/>
+      </div>
+      </>
```
Comment (thebearingedge) [inline client/pages/auth-modal.jsx:11]: You can remove this react fragment

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: client/pages/auth-modal.jsx (hunk 3 above)
Comment (yuliia-p) [reply client/pages/auth-modal.jsx:11]: Screen Shot 2022-09-16 at 9 59 34 AM

File: client/components/auth-form.jsx (hunk 2 above)
Comment (yuliia-p) [reply client/components/auth-form.jsx:60]: Screen Shot 2022-09-16 at 9 58 40 AM

**Comment by yuliia-p** [comment]
Screen Shot 2022-09-16 at 10 01 17 AM

File: list-of-databases.txt (hunk 1 above)
Comment (yuliia-p) [reply list-of-databases.txt]: Screen Shot 2022-09-16 at 10 01 17 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**